source/includes/crud/Delete.java

```diff
@@ -24,14 +27,26 @@ public static void main(String[] args) {
             MongoDatabase database = mongoClient.getDatabase("sample_mflix");
             MongoCollection<Document> collection = database.getCollection("movies");
 
-            Bson query = lt("imdb.rating", 1.9);
+            Bson deleteOneQuery = eq("title", "The Garbage Pail Kids Movie");
+
+            try {
+                // Deletes the first document that has a "title" value of "The Garbage Pail Kids Movie"
+                DeleteResult result = collection.deleteOne(deleteOneQuery);
+                System.out.println("DeleteOne document count: " + result.getDeletedCount());
+
+            // Prints a message if any exceptions occur during the operation
+            } catch (MongoException me) {
+                System.err.println("Unable to delete due to an error: " + me);
+            }
+
+            Bson deleteManyQuery = lt("imdb.rating", 1.9);
 
             try {
                 // Deletes all documents that have an "imdb.rating" value less than 1.9
-                DeleteResult result = collection.deleteMany(query);
+                DeleteResult result = collection.deleteMany(deleteManyQuery);
                 
                 // Prints the number of deleted documents
-                System.out.println("Deleted document count: " + result.getDeletedCount());
+                System.out.println("DeleteMany document count: " + result.getDeletedCount());
```

Comment:
S: same suggestion as above
```suggestion
                System.out.println("deleteMany() document count: " + result.getDeletedCount());
```